### PR TITLE
LicenseResolverTest: Test that excludes are applied properly

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: model/src/test/kotlin/utils/LicenseResolverTest.kt [hunk 1]
```diff
@@ -97,6 +97,17 @@ private fun OrtResult.getVcsProvenance(id: Identifier): Provenance? =
...
+private enum class ProvenanceType {
+    VCS,
+    SOURCE_ARTIFACT
+}
+
+private fun OrtResult.getProvenance(id: Identifier, type: ProvenanceType): Provenance? =
+    when (type) {
+        ProvenanceType.SOURCE_ARTIFACT -> getSourceArtifactProvenance(id)
+        else -> getVcsProvenance(id)
+    }!!
```
Comment (sschuberth) [inline model/src/test/kotlin/utils/LicenseResolverTest.kt:125]: You should now drop the `!!` here.

Comment (fviernau) [reply model/src/test/kotlin/utils/LicenseResolverTest.kt:125]: Done.

File: model/src/test/kotlin/utils/LicenseResolverTest.kt [hunk 2]
```diff
@@ -50,7 +50,10 @@ import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsType
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.PackageConfiguration
+import org.ossreviewtoolkit.model.config.PathExclude
```
Comment (sschuberth) [inline model/src/test/kotlin/utils/LicenseResolverTest.kt:53]: Commit message: s/LicenseResolver/LicenseResolverTest/

Comment (fviernau) [reply model/src/test/kotlin/utils/LicenseResolverTest.kt:53]: Done.

**Pull request merged**